release/models/wifi/openconfig-wifi-types.yang

```diff
@@ -59,6 +65,16 @@ module openconfig-wifi-types {
       domain.";
   }
 
+  typedef channels-type-6ghz {
```

Comment:
Consider simply adding the 6GHz channels to existing typedef to remain consistent w/ current modelling approach.

Reply:
ACK.